src/app/core/user.model.ts

```diff
@@ -2,10 +2,12 @@ export class FirebaseUserModel {
   image: string;
   name: string;
   provider: string;
+  email: string;
 
   constructor(){
```

Comment:
why there is constructor? where you create user instance? if you create instance then just user.ts , not model. And remove constructor and you can assign values when you create field.   image = '' etc. But i think it should be just interface, without constructor.